python/nano/tutorial/inference/pytorch/pytorch_quantization_inc.py

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
change to `pytorch_quantization.py`

Reply:
do we still need quantization using onnx and openvino? The contents are the same except for the accelerator argument.